test/unit/core/kernel/persistence/OntologyRdfsTest.php

```diff
@@ -28,20 +28,14 @@
 /**
  *
  */
-class OntologyMockTest extends GenerisTestCase
+class OntologyRdfsTest extends GenerisTestCase
```

Comment:
this is breaking the taoLti extension,  are we able to extend and not remove / rename so we are able to keep backward compatibility